src/platform/tigerlake/include/platform/lib/memory.h

```diff
@@ -51,6 +51,11 @@
 #define HOST_WIN_SIZE		0x00000008
 #define HOST_WIN_BASE(x)	(0x00071A00 + x * HOST_WIN_SIZE)
 
+/*
+ * size of HPSRAM system heap
+ */
+#define HEAPMEM_SIZE 0x40000
```

Comment:
@marcinszkudlinski D'oh, sorry, missed this in the review and focused on the IMR boot definitions. Please just drop this HEAPMEM_SIZE, we don't need it for legacy platforms. The definition in zephyr alloc.c "#define HEAPMEM_SIZE ((uint8_t *)_heap_sentry - heapmem)" handled the cAVS platforms.

Reply:
```
#elif CONFIG_ACE

/*
 * System heap definition for ACE is defined below.
 * It needs to be explicitly packed into dedicated section
 * to allow memory management driver to control unused
 * memory pages.
 */
__section(".heap_mem") static uint8_t __aligned(PLATFORM_DCACHE_ALIGN) heapmem[**HEAPMEM_SIZE**];
```
do we? For MTL we need to have limited heap size to keep HPSRAM free for loadable libraries

Reply:
@marcinszkudlinski I mean for older platform we size the heap based on linker script markers. So you'll need HEAPMEM_SIZE only for MTL. I guess we could reserve space to dynamic libs via linker script markers as well (just an idea).
This should still help to reduce conflicts.